### PR TITLE
docs(store-devtools): Added documentation for new extension options for Store Devtools

### LINKED ISSUE #1519: docs: Add new extension options for Store Devtools
## I would be willing to submit a PR for the docs :heart:

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

Store Devtools now supports persisting, locking, pausing, along with whitelisting and blacklisting of actions. These should be added to the Instrumentation docs.

### PR DESCRIPTION
**Pull request by wesleygrimes**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?
Updated documentation on Store Devtools Config page as per #1519 

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #1519 

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 1]
```diff
@@ -1,31 +1,87 @@
...
 
-## monitor
+### `monitor`
 
 function - the monitor function configuration that you want to hook.
 
-## actionSanitizer
+### `actionSanitizer`
+
+function - takes `action` object and id number as arguments, and should return `action` object.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/store-devtools/config.md]:
```suggestion
function - takes `action` object and id number as arguments, and should return an `action` object.
```

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 2]
```diff
@@ -1,31 +1,87 @@
...
+
+### `stateSanitizer`
+
+function - takes `state` object and index as arguments, and should return `state` object.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/store-devtools/config.md]:
```suggestion
function - takes `state` object and index as arguments, and should return a `state` object.
```

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 3]
```diff
@@ -1,31 +1,87 @@
...
+
+### `serialize`
+
+- options
+  - undefined - will use regular JSON.stringify to send data (it's the fast mode).
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/store-devtools/config.md]:
```suggestion
  - `undefined` - will use the regular `JSON.stringify` to send data
```

What is meant with  `(it's the fast mode)`?

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 4]
```diff
@@ -1,31 +1,87 @@
...
+  - false - will handle also circular references
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/store-devtools/config.md]:
```suggestion
  - `false` - will handle also circular references
```

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 5]
```diff
@@ -1,31 +1,87 @@
...
+  - true - will handle also date, regex, undefined, primitives, error objects, symbols, maps, sets and functions
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/store-devtools/config.md]:
```suggestion
  - `true` - will handle also dates, regexes, undefined, primitives, error objects, symbols, maps, sets and functions
```

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 6]
```diff
@@ -1,31 +1,87 @@
...
+  - object - which contains date, regex, undefined, nan, infinity, error, symbol, map, set and function keys. For each of them you can indicate if to include (by setting as true). For function key you can also specify a custom function which handles serialization
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/store-devtools/config.md]:
```suggestion
  - object - which contains `date`, `regex`, `undefined`, `nan`, `infinity`, `error`, `symbol`, `map`, `set` and `function` keys. For each of them, you can indicate if they have to be included by setting them to `true`. For function keys, you can also specify a custom function which handles serialization.
```

Something like this? 🤔

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 7]
```diff
@@ -1,31 +1,87 @@
...
+
+### `logOnly`
 
 boolean - connect to the Devtools Extension in log-only mode. Default is `false` which enables all extension [features](https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md#features).
 
-## name
+### `name`
 
-string - the instance name to be showed on the monitor page. Default value is _NgRx Store DevTools_.
+string - the instance name to be showed on the monitor page. Default value is NgRx Store DevTools.
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/store-devtools/config.md]:
```suggestion
string - the instance name to show on the monitor page. Default value is _NgRx Store DevTools_.
```

File: projects/ngrx.io/content/guide/store-devtools/config.md (hunk 3 above)
Comment (timdeschryver) [reply projects/ngrx.io/content/guide/store-devtools/config.md]: I agree 👍

**Comment by wesleygrimes** [comment]
@timdeschryver latest commit should have all the changes requested. Let me know if you need anything else.

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 8]
```diff
@@ -1,31 +1,87 @@
...
 
-function - takes `action` object and id number as arguments, and should return `action` object back.
+### `features`
 
-## stateSanitizer
+configuration object - containing properties for features than can be enabled or disabled in the browser extension Redux DevTools. These options are passed through to the browser extension verbatim. By default, all features are enabled. For more information visit the [Redux DevTools Docs](https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md#features)
 
-function = takes `state` object and index as arguments, and should return `state` object back.
+```typescript
+features: {
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/store-devtools/config.md]: Fix indention for spacing

File: projects/ngrx.io/content/guide/store-devtools/config.md [hunk 9]
```diff
@@ -1,31 +1,87 @@
...
+    dispatch: true, // dispatch custom actions or action creators
+    test: true // generate tests for the selected actions
+},
+```
 
-## serialize
+## Example Object as provided in module imports
 
-false | configuration object - Handle the way you want to serialize your state, [more information here](https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md#serialize).
+<code-example header="app.module.ts">
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/store-devtools/config.md]: Fix indention for spacing

**Pull request merged**